shell/platform/android/io/flutter/embedding/engine/FlutterShellArgs.java

```diff
@@ -43,6 +43,8 @@
   public static final String ARG_DUMP_SHADER_SKP_ON_SHADER_COMPILATION = "--dump-skp-on-shader-compilation";
   public static final String ARG_KEY_VERBOSE_LOGGING = "verbose-logging";
   public static final String ARG_VERBOSE_LOGGING = "--verbose-logging";
+  public static final String ARG_KEY_OBSERVATORY_PORT = "observatory-port";
+  public static final String ARG_OBSERVATORY_PORT = "--observatory-port=";
```

Comment:
Should this "=" be here?

Reply:
It either has to be here or in the concatenation below.  I don't have a strong preference - this is the only arg like this right now in here.